### PR TITLE
Development

### PR DESCRIPTION
**Pull request by Shamanou**
Bugfix: 

fixed a regex parsing bug in the parsing of manually inserted urls.
Now is there are multiple colons, it will only apply the regex to strings surrunding the first colon

### CODE REVIEW

**Review by nunogit — COMMENTED** [review]
Looks good. Since regular expressions can become a bit cryptic as they grow I would just add a couple of comments to make what you're doing more explicit.

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js [hunk 1]
```diff
@@ -99,7 +99,11 @@ RdfPrefixesManager.prototype._hasPrefix = function(name){
 * Some utility functions
 */
 RdfPrefixesManager.isPrefixedQname = function(qname){
-	return qname.match(/[_a-zA-Z][-_a-zA-Z0-9]*:($|([_a-zA-Z][-_a-zA-Z0-9]*))$/);
+        if ((qname.split(':').length -1) < 3){
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js]: So what happens if a URL contains more than 1 colon in the path?

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js [hunk 2]
```diff
@@ -99,7 +99,11 @@ RdfPrefixesManager.prototype._hasPrefix = function(name){
...
+            return qname.match(/[_a-zA-Z][-_a-zA-Z0-9]*:($|([_a-zA-Z][-_a-zA-Z0-9]*))$/);
+        } else{
+            return (qname.split(':')[0]+ ':' +  qname.split(':')[1]).match(/[_a-zA-Z][-_a-zA-Z0-9]*:($|([_a-zA-Z][-_a-zA-Z0-9]*))$/);
```
Comment (kburger) [inline extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js]: The regex is repeated twice, maybe you can capture it in a variable to make it a bit more readable?

**Review by kburger — COMMENTED** [review]
Without any prior knowledge on the problem nor the possible values, these 2 things come to mind.

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js (hunk 1 above)
Comment (Shamanou) [reply extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js]: Then it splits the string by colon and then checks with the regex but only against the two first values in the first list joined by a colon. The regex is checking for prefixes i dont think they need the full string

File: extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js (hunk 2 above)
Comment (Shamanou) [reply extensions/fairifier-rdf-extension/scripts/rdf-schema-vocabulary-manager.js]: sure will do

**Pull request merged**